packages/office-addin-debugging/src/defaults.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as usageData from "office-addin-usage-data";
+
+// Usage data defaults
+export const usageDataProjectName: string = "office-addin-debugging";
+export const sendUsageData: boolean = usageData.groupNameExists(usageData.groupName) && usageData.readUsageDataLevel(usageData.groupName) === usageData.UsageDataLevel.on;
```

Comment:
Why does sendUsageData and usageDataOptions need to be defined here. Can't this stuff come directly from office-addin-usage-data? Are there any plans to improve/fix that API rather than put a lot of redundant stuff in each package?

Reply:
Agreed we shouldn't need to define another sendUsageData method in each of the defaults.  We can just create a new exported method (if one doesn't already exist) in office-addin-usage data that wraps groupNameExists and readUsageDataLevel and returns true or false